Lombiq.OrchardCoreApiClient/ApiClient.cs

```diff
@@ -6,20 +6,26 @@
 using System;
 using System.Collections.Generic;
 using System.Globalization;
+using System.Net.Http;
 using System.Net.Http.Headers;
 using System.Threading.Tasks;
 
 namespace Lombiq.OrchardCoreApiClient;
 
 public class ApiClient
 {
-    private readonly ApiClientSettings _apiClientSettings;
     private Lazy<IOrchardCoreApi> LazyOrchardCoreApi => new(() => GetOrchardCoreApi(_apiClientSettings.DefaultTenantUri));
-    private DateTime _expirationDateUtc = DateTime.MinValue;
-    private Token _tokenResponse;
+    protected readonly ApiClientSettings _apiClientSettings;
+    protected readonly HttpClient _httpClient;
+    protected DateTime _expirationDateUtc = DateTime.MinValue;
+    protected Token _tokenResponse;
     public IOrchardCoreApi OrchardCoreApi => LazyOrchardCoreApi.Value;
 
-    public ApiClient(ApiClientSettings apiClientSettings) => _apiClientSettings = apiClientSettings;
+    public ApiClient(ApiClientSettings apiClientSettings, HttpClient httpClient = null)
+    {
+        _apiClientSettings = apiClientSettings;
+        _httpClient = httpClient ?? new HttpClient();
```

Comment:
When `ApiClient` owns the `HttpClient` instance, it should create it with the factory: https://learn.microsoft.com/en-us/dotnet/core/extensions/httpclient-factory